### ISSUE TITLE
Lack of documentation

### ISSUE THREAD

**Issue by SpooikypokDev** [OxBrowse/EXPLORE_File#3]
There should be a wiki or a documentation where every contributor describes how the function it made works, or at least is supposed to work. I have to know how the browser is supposed to behave to know how i should treat a function, or to know if something is an undeclared bug or not.

**Comment by card100** [comment]
Sure, but we should first get it working/done.

**Comment by SpooikypokDev** [comment]
Yes, but for instance, I'm trying to develop, but I don't know how well it works on Windows, I don't know if what I'm seeing is an actual bug or the result of the stupid behavior of mac OS. So, what have we done so far? Is there a particular syntax for file uris? Etc, etc.

**Comment by card100** [comment]
Ok, true.  Sure.
